charts/fluvio/charts/crds/templates/crd_topic.yaml

```
@@ -0,0 +1,199 @@
+                        count:
+                          type: integer
+                          minimum: 0
+                        age:
+                          type: string
+                          nullable: true
+                system:
+                  type: boolean
+      subresources:
+          status: {}
+      additionalPrinterColumns:
+          - name: Partitions
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)